etc/kayobe/kolla.yml

```diff
@@ -104,6 +104,17 @@ kolla_sources:
     type: git
     location: "{{ kolla_bifrost_source_url }}"
     reference: "{{ kolla_bifrost_source_version }}"
+  cloudkitty-base:
+    # Backport of NOTNUMBOOL: https://review.opendev.org/c/openstack/cloudkitty/+/774084
```

Comment:
Not sure if I should bother maintaining these comments here as they may get out of sync with the branch. Thoughts?

Reply:
They could go stale, maybe leave them out.